WooCommerce/Classes/Authentication/Navigation Exceptions/RoleErrorViewModel.swift

```diff
@@ -14,6 +17,10 @@ final class RoleErrorViewModel {
         .incorrectRoleError
     }
 
+    var onSuccess: (() -> Void)?
+
+    var onDeauthenticationRequest: (() -> Void)?
```

Comment:
Since this is an API meant to be consumed It would be good to add some docs too!

Reply:
Noted, will add some docs soon! 👍